### PR TITLE
fortio 0.9.0 (new formula)

### PR DESCRIPTION
**Pull request by ymesika**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/fortio.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "-X istio.io/fortio/version.tag=#{tag} "\
+        "-X \"istio.io/fortio/version.buildInfo=#{build_info}\""
+      system "go", "build", "-a", "-ldflags", link_flags.to_s
+      bin.install "fortio"
+      lib.install Dir["ui/static", "ui/templates"]
+    end
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/fortio --version")
```
Comment (ldemailly) [inline Formula/fortio.rb]: should be `fortio version -s` to get 0.9.0 vs the full build info

how/where is version defined?

File: Formula/fortio.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+class Fortio < Formula
+  desc "HTTP and gRPC load testing and visualization tool and server"
+  homepage "https://github.com/istio/fortio"
+  url "https://github.com/istio/fortio.git",
+      :using => :git,
+      :tag => "v0.9.0",
+      :revision => "e50943e8e525197f36f9b4f81464615c063a0a65"
```
Comment (ldemailly) [inline Formula/fortio.rb]: anyway to get the latest release/tag at the time of build? or do we need to update this for every release ? `latest_release` in git points to the latest good tag

File: Formula/fortio.rb (hunk 1 above)
Comment (ymesika) [reply Formula/fortio.rb]: Seems like the `assert_match` ignores anything after a space character which comes right after the short version and treats it as `0.9.0` although it's the long version.
I.e. test pass as it is now.

I printed the `version` variable and it's `0.9.0`. I guess it's extracted from the tag.

File: Formula/fortio.rb (hunk 2 above)
Comment (ymesika) [reply Formula/fortio.rb]: We can but then you:
* Experience a crash in brew build which seems like version in the tag
* Won't pass `brew audit --strict` as it expects `revision` too.
* Won't have the ability to install an earlier version and

File: Formula/fortio.rb (hunk 1 above)
Comment (fxcoudert) [inline Formula/fortio.rb]: We need a test that exercises the some of the functionality of the app.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by ymesika** [comment]
Replaced with a new PR: #26653

**Comment by ldemailly** [comment]
for future reference, you can usually use git reflog + git reset + push to undo errors and update a PR even after you added bad commits

**Pull request closed without merging**